BlueShift-iOS-SDK/BlueShiftAppDelegate.m

```diff
@@ -659,7 +668,7 @@ - (void)handleCarouselPushForCategory:(NSString *)categoryName usingPushDetailsD
             NSDictionary *selectedItem = [carouselItems objectAtIndex:index];
             NSString *urlString = [selectedItem objectForKey: kPushNotificationDeepLinkURLKey];
             NSURL *url = [NSURL URLWithString:urlString];
-            [pushDetails setValue:urlString forKey:kPushNotificationDeepLinkURLKey];
+            [pushDetails setValue:urlString forKey:kNotificationURLElementKey];
```

Comment:
this key change is not affected existing implementation, right?

Reply:
Yes, this will not affect the rest implementation. This is an exceptional case for the carousel notification and is handled in the setupPushNotificationDeeplink method.